amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -8,6 +8,7 @@ import { RouteComponentProps } from 'react-router';
 
 import SearchBar from './SearchBar';
 import SearchList from './SearchList';
+import LoadingSpinner from 'components/common/LoadingSpinner';
```

Comment:
This might be out of scope, but the LoadingSpinner has a fixed height and width that may not look great in all places. Maybe look into how to make it smaller/bigger depending on the context.

Reply:
I think this is probably out of scope for this PR, will file a follow up task